adapta/storage/delta_lake/README.md

```diff
@@ -22,6 +23,59 @@ adls_path = AdlsGen2Path.from_hdfs_path('abfss://container@account.dfs.core.wind
 # get Iterable[pandas.DataFrame]
 batches = load(azure_client, adls_path, batch_size=1000)
 ```
+
+### For AWS Simple Storage Service (S3) or S3-Compatible Storage
+
+```python
+import os
+from adapta.security.clients import AwsClient
+from adapta.security.clients.aws._aws_credentials import EnvironmentAwsCredentials
+from adapta.storage.delta_lake import load
+
+# Set up environment variables
+os.environ["PROTEUS__AWS_ACCESS_KEY_ID"] = minio_access_key_id
+os.environ["PROTEUS__AWS_SECRET_ACCESS_KEY"] = minio_secret_key
+os.environ["PROTEUS__AWS_REGION"] = "eu-central-1"
+os.environ["PROTEUS__AWS_ENDPOINT"] = "http://my-endpoint.com"
```

Comment:
```suggestion
os.environ["PROTEUS__AWS_ENDPOINT"] = "http://example.com"
```

Reply:
Thank you for the suggestion 🙂 much more general indeed